.github/workflows/ci-main.win.yml

```diff
@@ -11,18 +11,50 @@ jobs:
     runs-on: windows-2019
 
     steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v2
+      - name: Checkout source code
+        uses: actions/checkout@v2
 
-      - name: Setup Node.js environment
-        uses: actions/setup-node@v2
+      - name: Setup Yarn environment
+        uses: DerYeger/yarn-setup-action@v1.0.1
         with:
-          node-version: '14'
-          # Set this option if you want the action to check for the latest available version that satisfies the version spec
-          check-latest: true
+          node-version: 16
 
-      - name: Build the extension and run tests
-        run: pwsh -NonInteractive -NoProfile -NoLogo scripts/ci.ps1
+      - name: Build the extension
+        run: yarn compile-production
+
+      - name: Lint
+        run: yarn lint
+
+      - name: Install CMake/Ninja
+        uses: lukka/get-cmake@v3.18.3
```

Comment:
did we choose this from GitHub Actions based on the number of stars?

Reply:
yes